src/js/services/bootstrap.js

```diff
@@ -54,15 +54,17 @@ export default App.extend({
     return this.currentOrg;
   },
   getOrgSetting(settingName) {
-    return this.getCurrentOrg().getSetting(settingName);
+    const setting = this.getCurrentOrg().getSetting(settingName);
+    if (!setting) return;
```

Comment:
I am not sure if we have anywhere in the tests that a setting is request that doesn't exist, so this might need coverage.. or maybe just an istanbul ignore.

Reply:
Doesn't appear that happens in the tests: https://coveralls.io/builds/56308171/source?filename=src%2Fjs%2Fservices%2Fbootstrap.js#L58